tasks/install.yml

```diff
@@ -1,58 +1,15 @@
 ---
-- name: https apt transport for docker repository
-  apt:
-    name: apt-transport-https
-  tags:
-    - dokku
-    - docker-repo
-
-- name: gnupg-curl for docker gpgkey
-  apt:
-    name: gnupg-curl
-    state: '{{ "present" if ansible_distribution_release in
-               [ "wheezy", "jessie", "precise", "trusty", "xenial" ]
-               else "absent" }}'
-  tags:
-    - dokku
-    - docker-repo
-
-- name: apt.docker.com key
-  apt_key:
-    id: "{{ docker_apt_key_sig }}"
-    keyserver: "{{ docker_apt_key_url }}"
-    state: present
-  tags:
-    - dokku
-    - docker-repo
-
-- name: download.docker.com key
-  apt_key:
-    id: "{{ docker_download_key_sig }}"
-    keyserver: "{{ docker_download_key_url }}"
-    state: present
-  tags:
-    - dokku
-    - docker-repo
-
 - name: packagecloud dokku apt key
   apt_key:
     url: https://packagecloud.io/dokku/dokku/gpgkey
   tags:
     - dokku
 
-- name: docker repo
-  apt_repository:
-    filename: docker
-    repo: 'deb [arch=amd64] https://download.docker.com/linux/ubuntu {{ ansible_lsb.codename|lower }} stable'
-    state: present
-  tags:
-    - dokku
-    - docker-repo
-
 - name: dokku repo
   apt_repository:
     filename: dokku
-    repo: 'deb https://packagecloud.io/dokku/dokku/ubuntu/ {{ ansible_lsb.codename|lower }} main'
+    repo: 'deb https://packagecloud.io/dokku/dokku//{{ ansible_distribution|lower }} {{ ansible_distribution_release }} main'  # yamllint disable-line rule:line-length
```

Comment:
I think the trailing slash was changed incorrectly here.

Reply:
Thanks for spotting.
Interesting that this still seemed to work...